text_generation/causal_lm/cpp/continuous_batching/python/tests/models/nightly

```diff
@@ -0,0 +1,145 @@
+hf-internal-testing/tiny-random-BigBirdForPreTraining
+hf-internal-testing/tiny-random-transfo-xl
+hf-internal-testing/tiny-random-IBertForQuestionAnswering
+hf-internal-testing/tiny-random-CanineForTokenClassification
+hf-internal-testing/tiny-random-SplinterForQuestionAnswering
+hf-internal-testing/tiny-random-PegasusXForConditionalGeneration
+hf-internal-testing/tiny-random-XmodForCausalLM
+hf-internal-testing/tiny-random-BigBirdForQuestionAnswering
+hf-internal-testing/tiny-random-MT5ForSequenceClassification
+hf-internal-testing/tiny-random-MarianForCausalLM
+hf-internal-testing/tiny-random-Data2VecTextForQuestionAnswering
+hf-internal-testing/tiny-random-SpeechT5Model
+hf-internal-testing/tiny-random-Data2VecAudioForSequenceClassification
+hf-internal-testing/tiny-random-MaskFormerModel
+hf-internal-testing/tiny-random-M2M100ForConditionalGeneration
+hf-internal-testing/diffusers-images
+hf-internal-testing/tiny-random-ViTMSNModel
+hf-internal-testing/tiny-random-MegatronBertForSequenceClassification
+hf-internal-testing/tiny-random-MobileNetV2Model
+hf-internal-testing/tiny-random-DistilBertForQuestionAnswering
+hf-internal-testing/tiny-random-PerceiverModel
+hf-internal-testing/tiny-random-M2M100Model
+hf-internal-testing/tiny-random-vision-encoder-decoder
+hf-internal-testing/tiny-random-XLNetForTokenClassification
+hf-internal-testing/tiny-random-SwitchTransformersForConditionalGeneration
+hf-internal-testing/tiny-electra
+hf-internal-testing/tiny-random-GLPNModel
+hf-internal-testing/tiny-random-NezhaForPreTraining
+hf-internal-testing/test-two-configs
+hf-internal-testing/tiny-random-Dinov2ForImageClassification
+hf-internal-testing/tiny-bert-pt-safetensors-msgpack
+hf-internal-testing/tiny-random-owlvit
+hf-internal-testing/tiny-random-VitsModel
+hf-internal-testing/tiny-random-MT5ForTokenClassification
+hf-internal-testing/tiny-random-xlnet
+hf-internal-testing/tiny-random-MusicgenMelodyForCausalLM
+hf-internal-testing/tiny-random-RegNetModel
+hf-internal-testing/tiny-random-MarkupLMForTokenClassification
+hf-internal-testing/tiny-random-Speech2TextModel
+hf-internal-testing/llama-code-tokenizer
+hf-internal-testing/tiny-random-T5ForSequenceClassification
+hf-internal-testing/tiny-random-BartForConditionalGeneration
+hf-internal-testing/tiny-random-PegasusModel
+hf-internal-testing/tiny-random-TransfoXLModel
+hf-internal-testing/tiny-random-BigBirdForMultipleChoice
+hf-internal-testing/tiny-random-PLBartForCausalLM
+hf-internal-testing/tiny-random-SwinBackbone
+hf-internal-testing/tiny-random-electra
+hf-internal-testing/tiny-random-FNetForMaskedLM
+hf-internal-testing/tiny-random-PersimmonForSequenceClassification
+hf-internal-testing/tiny-random-MobileViTV2ForImageClassification
+hf-internal-testing/tiny-random-XGLMForCausalLM
+hf-internal-testing/tiny-random-SEWForCTC
+hf-internal-testing/tiny-random-UMT5ForSequenceClassification
+hf-internal-testing/tiny-random-PersimmonModel
```

Comment:
can we keep only CausalLM models https://huggingface.co/hf-internal-testing?search_models=CausalLM as agreed before ?

Reply:
Yes, done.